sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/appconfiguration/AzureAppConfigurationProperties.java

```diff
@@ -19,6 +19,7 @@
     private String connectionString;
     private ConfigurationServiceVersion serviceVersion;
 
+    @Override
```

Comment:
Why is this also override?

Reply:
Added the interface `ConnectionStringAware`